src/main/scala/uk/gov/nationalarchives/consignmentexport/BagMetadata.scala

```diff
@@ -18,7 +18,14 @@ class BagMetadata(keycloakClient: KeycloakClient)(implicit val logger: SelfAware
     private def isStringNullOrEmpty(s: String): Boolean = s == null || s.trim.isEmpty
 
     def isUserRepresentationComplete: Boolean = {
-      !isStringNullOrEmpty(value.getFirstName) && !isStringNullOrEmpty(value.getLastName)
+      !isStringNullOrEmpty(value.getFirstName) && !isStringNullOrEmpty(value.getLastName) && !isStringNullOrEmpty(value.getEmail)
```

Comment:
Maybe these could be stacked on top of one another like

     !isStringNullOrEmpty(value.getFirstName) &&
      !isStringNullOrEmpty(value.getLastName) &&
      !isStringNullOrEmpty(value.getEmail)
?

Reply:
Refactored this a little bit to get rid of the negation `!` on the `isStringNullOrEmpty` function to make it clearer to read.